include/micm/process/process_set.hpp

```diff
@@ -304,15 +303,15 @@ namespace micm
             d_rate_d_ind[i_cell] = v_rate_constants[offset_rc + i_rxn * L + i_cell];
           for (std::size_t i_react = 0; i_react < number_of_reactants_[i_rxn]; ++i_react)
           {
-            if (i_react == i_ind)
+             if (i_react == i_ind)
```

Comment:
Is this space intended?

Reply:
No. Just the format difference. I removed the space.